src/storage/rdb.cc

```diff
@@ -728,6 +731,8 @@ Status RDB::SaveObjectType(const RedisType type) {
     robj_type = RDBTypeSet;
   } else if (type == kRedisZSet) {
     robj_type = RDBTypeZSet2;
+  } else if (type == kRedisBitmap) {
+    robj_type = RDBTypeBitmap;
```

Comment:
As mentioned in the above comment, we can convert the robj_type to RDBTypeString here.